package.json

```diff
@@ -21,12 +21,12 @@
     "rfc 2045",
     "rfc 2047"
   ],
-  "author": "Joran Dirk Greef",
+  "author": "Joran Dirk Greef", "Vlad Lasky"
```

Comment:
looks like a syntax error

Reply:
Why change everything to vlasky? Just patch the source.

Reply:
I think you can add your name under the [contributors](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors) array

Reply:
@jchook I did a force push immediately after I noticed that mistake. I thought I did it quickly, but apparently not quick enough :-). Please pull the code again.